packages/providers/inpage-providers-hub/src/floatingButton/index.tsx

```
@@ -0,0 +1,746 @@
+              fontWeight: '500',
+            }}
+          >
+            {securityInfo?.dapp?.logo ? (
+              <img
+                src={securityInfo?.dapp?.logo}
+                style={{
+                  height: '24px',
+                  width: '24px',
+                  borderRadius: '4px',
+                }}
+              />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)